src/main/plugins/org.dita.base/plugin.xml

```diff
@@ -51,6 +51,10 @@ See the accompanying LICENSE file for applicable license.
   <!-- Deprecated since 2.1 -->
   <extension-point id="depend.preprocess.copy-subsidiary.pre" name="Copy subsidiary pre-target"/>
   <extension-point id="depend.preprocess.post" name="Preprocessing post-target"/>
+  <extension-point id="depend.preprocess2.maps.pre" name="Map-first pre-processing maps pre-target"/>
```

Comment:
@infotexture The copy needs to be better 😄

Reply:
Let's follow the pattern we use in the docs for the other [Pre-processing extension points](https://www.dita-ot.org/dev/extension-points/plugin-extension-points-pre-processing):

```suggestion
  <extension-point id="depend.preprocess2.maps.pre" name="Runs an Ant target before pre-processing maps"/>
```